<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogimen</class>
 <widget class="QDialog" name="Dialogimen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>916</width>
     <height>791</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Qanelas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Employés</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>911</width>
       <height>651</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Consulter la liste des Employés</string>
      </attribute>
      <widget class="QLabel" name="label_ajoutempl">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>0</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>891</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background :rgb(145, 137, 255)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Qanelas</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
selection-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Liste des employés</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>891</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>420</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>position</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>420</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>420</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>420</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NOM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PRENOM</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>420</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>trier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_19">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>470</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Imprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Nouvel_Employé</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>110</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sakkal Majalla</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ajoutempl_3">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>100</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="image">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>161</width>
         <height>161</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>220</y>
         <width>281</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>ID_Sociale </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit_IDsociale"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Numero Securite sociale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_numsecurite"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Numero CIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_cin"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>220</y>
         <width>291</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Status sociale</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Sexe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_sexe"/>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Etat civile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_etatcivil"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>430</y>
         <width>271</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>Contacte_informations</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>adresse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_adresse"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Telephone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Telephone"/>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>E_mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Email"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_position">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>440</y>
         <width>301</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string>Position</string>
       </property>
       <widget class="QComboBox" name="comboBox_position">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>221</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>43</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>80</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenom">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>80</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ID">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>120</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>130</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>160</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sakkal Majalla</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>160</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>190</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter image</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Congés</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>911</width>
       <height>731</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Consulter la liste des congés</string>
      </attribute>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>901</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background :rgb(145, 137, 255)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Qanelas</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
selection-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Liste des Congés</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="tableConge">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>111</y>
         <width>801</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>430</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>430</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>430</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>430</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>idconge</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>type</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>datedebut</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>430</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_5">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>430</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATE_DEBUT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TYPE</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_17">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>430</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>trier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_18">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>480</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Imprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Nouvel_congé</string>
      </attribute>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>80</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sakkal Majalla</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_icopcon">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>80</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Dates du conge</string>
       </property>
      </widget>
      <widget class="Line" name="line_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>351</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_DateDebutConge">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date début conge :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_DateFinConge">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date fin conge :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_TypeConge">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type de conge :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Informations supplémentaires</string>
       </property>
      </widget>
      <widget class="Line" name="line_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>351</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_CR">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>390</y>
         <width>291</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_CE">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>550</y>
         <width>291</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_commentaireResponsable_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>520</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Commentaire employé : </string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxtype">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>260</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEditdebut">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>180</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_2_datefin">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>220</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_commentaireResponsable_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>commentaireResponsable:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_idConge">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>70</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_DateDebutConge_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_congé</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>140</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>140</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sakkal Majalla</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
